axon-server-connector/src/main/java/org/axonframework/axonserver/connector/command/AxonServerCommandBus.java

```diff
@@ -490,6 +505,21 @@ public Builder executorServiceBuilder(ExecutorServiceBuilder executorServiceBuil
             return this;
         }
 
+        /**
+         * Sets the {@link CommandLoadFactorProvider} which provides the load factor values for all commands this
+         * client can handle. The load factor values are sent to AxonServer during command subscription. AxonServer
+         * uses these values to balance the dispatching of commands among the client instances.
+         *
```

Comment:
Nit: It would be beneficial if the javadoc of this method describes what the default `CommandLoadFactorProvider` is.